lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/core/impl/qm/XwikiBridgeInterfaceRestResource.java

```diff
@@ -22,51 +22,61 @@
 import org.xwiki.component.annotation.Component;
 import org.xwiki.rest.XWikiRestComponent;
 
-import eu.learnpad.exception.LpRestException;
-import eu.learnpad.exception.impl.LpRestExceptionImpl;
+import eu.learnpad.exception.impl.LpRestExceptionXWikiImpl;
 import eu.learnpad.qm.BridgeInterface;
 import eu.learnpad.core.rest.RestResource;
 
+/*
+ * The methods inherited form the BridgeInterface in this
+ * class should be implemented as a REST invocation
+ * toward the BridgeInterface binded at the provided URL
+ */
 @Component
```

Comment:
Btw waiting your reposnse I am reading again "http://platform.xwiki.org/xwiki/bin/view/DevGuide/WritingComponents" in order to get some inspiration.